Makefile

```diff
@@ -80,7 +81,10 @@ welcome:
 	@printf "\033[36m               |___/|_|                  \n"
 	@printf "\033[m\n"
 
-install: node_modules
+yarn_available:
```

Comment:
this seems like an odd name for a make target. it's almost as if it should return some boolean indication, but it's really just forcing a dependency. maybe something like `demand_yarn` or `yarn_check` would fit better